scripts/prerelease

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eufo pipefail
+
+npm outdated --long
+npm whoami
+npm run build
+transcribe --url 'https://github.com/fluture-js/fluture-node/blob/master/{filename}#L{line}' index.mjs > README.md
```

Comment:
This step might not be needed. I copied it for another project. Let me see how I can hook into the release cycle such that I can do checks like `npm outdated` and `npm whoami`, and run the `npm run build`.